src/t8_forest/t8_forest_vtk.cxx

```diff
@@ -114,10 +114,10 @@ typedef int (*t8_forest_vtk_cell_data_kernel) (t8_forest_t forest, const t8_loci
 
 #define T8_FOREST_VTK_QUADRATIC_ELEMENT_MAX_CORNERS 20
 /** Lookup table for number of nodes for curved eclasses. */
-const int t8_curved_eclass_num_nodes[T8_ECLASS_COUNT] = { 1, 3, 8, 6, 20, 10, 15, 13 };
+[[maybe_unused]] const int t8_curved_eclass_num_nodes[T8_ECLASS_COUNT] = { 1, 3, 8, 6, 20, 10, 15, 13 };
```

Comment:
Used here: https://github.com/DLR-AMR/t8code/blob/aa2b436a07bad4066b35be0742a036dd3975a3e5/src/t8_forest/t8_forest_vtk.cxx#L172

Reply:
Yeah I tried without vtk on my machine, which explains why I got an unused variable warning...

Reply:
Can you please remove those markers again?

Reply:
Done